src/extension.ts

```diff
@@ -312,6 +312,7 @@ export function activate(context: vscode.ExtensionContext): void {
             vscode.window.showInformationMessage(`File logging has already been enabled for ${siteTreeItem.client.fullName}.`);
         }
     });
+    registerCommand('appService.OpenCosmosDBExtension', () => vscode.commands.executeCommand('extension.open', 'ms-azuretools.vscode-cosmosdb'));
 }
 
```

Comment:
VS Code _just_ added the 'extension.open' command, right? We need to handle the situation when that command doesn't exist for people on older versions of VS Code.

There's two options:
1. Bump the minimum VS Code version here: https://github.com/Microsoft/vscode-azureappservice/blob/master/package.json#L10
1. Have backup logic if the command doesn't exist (maybe just open the browser?)

I vote option 2 because this doesn't seem like a good enough reason to bump the min VS Code version. Although don't spend a lot of time on the "backup logic" because it really is just "backup" logic that shouldn't happen very often